CorsixTH/Src/iso_fs.cpp

```diff
@@ -359,15 +359,21 @@ void iso_filesystem::clear() {
   delete[] error;
   error = nullptr;
   files.clear();
+  if (raw_file) fclose(raw_file);
```

Comment:
Also set it to `nullptr` to avoid double closing (one time calling `clear` in some usage code, and one time while destroying the object)